### PR TITLE
Schema change: Affiliations.txt

### PR DESCRIPTION
**Pull request by cyhuang01**

### CODE REVIEW

File: academic-services/graph/reference-data-schema.md [hunk 1]
```diff
@@ -85,6 +89,17 @@ Column # | Name | Type | Note
...
 
+## EntityRelatedEntities.txt
+
+Column # | Name | Type | Note
+--- | --- | --- | ---
+1 | EntityId | long |
+2 | EntityType | string | af: Affiliation <br> j: Journal <br> c: Conference
+3 | RelatedEntityId | long |
+4 | RelatedEntityType | string | af: Affiliation <br> j: Journal <br> c: Conference
+5 | RelatedType | int | 0: Two entities are similar if they appear on the same paper <br> 1: Two entities are similar if they have common coauthors <br> 2: Two entities are similar if they are co-cited by others <br> 3: Two entities are similar if they have common fields of study <br> 4: Two entities are similar if they appear in the same venue <br> 5: Entity A is similar to entity B if A cites B <br> 6: Entity A is similar to entity B if A is cited by B
```
Comment (DarrinEide) [inline academic-services/graph/reference-data-schema.md]: Shouldn't "RelatedType" be "RelationshipType"?

Comment (DarrinEide) [inline academic-services/graph/reference-data-schema.md]: Suggestion:
A cleaner way of approaching these nested table enumerations would simply be to create a new table in the parent tables section with a ### sub-heading, e.g.:
## EntityRelatedEntities.txt
_< main table with link to RelationshipType >_

### RelationshipType enumeration
_< table with enumeration values >_

I like the approach as it lets us directly link to the enumeration definition from other places (i.e. blog posts, support emails, etc.)

Comment (cyhuang01) [reply academic-services/graph/reference-data-schema.md]: I agree. But it is RelatedType in the script. I will keep it as is.

Comment (cyhuang01) [reply academic-services/graph/reference-data-schema.md]: Thanks for the suggestion. But I am concerned reader might confuse RelationshipType is another file in the MAG dataset.

**Pull request merged**